appointment/views.py

```diff
@@ -68,7 +68,6 @@ def meeting_list(request, year, month, day):
 
 def detail(request, meeting_id):
     """Render to meeting's detail page."""
-    # meetings = Meeting.objects.get(id=meeting_id)
     meetings = get_object_or_404(Meeting, pk=meeting_id)
```

Comment:
nice change!! for handle 404 error when the user tries to access does not exist meeting